config/tsa/tsa/300-tsa.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tsa
+  namespace: tsa-system
+---
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  namespace: tsa-system
+  name: tsa
+spec:
+  template:
+    metadata:
+      annotations:
+        autoscaling.knative.dev/min-scale: "1"
+    spec:
+      serviceAccountName: tsa
+      containers:
+      - name: tsa
+        image: ko://github.com/sigstore/timestamp-authority/cmd/timestamp-server/
```

Comment:
can we use the GHCR image from https://github.com/sigstore/timestamp-authority/pkgs/container/timestamp-server? just need to find which is the correct container from the hash

Reply:
Yeah, I didn't see them, so I created an issue, which was resolved. Fixed.
https://github.com/sigstore/timestamp-authority/issues/126